{"links": {"self": {"href": "data/repositories/levitsky/pyteomics/pullrequests/25/comments/75958427.json"}, "code": {"href": "https://api.bitbucket.org/2.0/repositories/levitsky/pyteomics/diff/mobiusklein/pyteomics:386b5af23730..3ff1d5dd257c?path=pyteomics%2Fmass%2Funimod.py"}, "html": {"href": "#!/levitsky/pyteomics/pull-requests/25/_/diff#comment-75958427"}}, "parent": {"id": 75958289, "links": {"self": {"href": "data/repositories/levitsky/pyteomics/pullrequests/25/comments/75958289.json"}, "html": {"href": "#!/levitsky/pyteomics/pull-requests/25/_/diff#comment-75958289"}}}, "deleted": false, "pullrequest": {"type": "pullrequest", "id": 25, "links": {"self": {"href": "data/repositories/levitsky/pyteomics/pullrequests/25.json"}, "html": {"href": "#!/levitsky/pyteomics/pull-requests/25"}}, "title": "Update for 4.0"}, "content": {"raw": "I have this same exact code in `psims.controlled_vocabulary.unimod` that I have test on Py2.7 and 3.6 on Travis-CI, and it worked fine there too. I put a virtualenv Py3.6 on my university\u2019s cluster to run some tests while writing the other changes shown here, and this appeared out of nowhere.", "markup": "markdown", "html": "<p>I have this same exact code in <code>psims.controlled_vocabulary.unimod</code> that I have test on Py2.7 and 3.6 on Travis-CI, and it worked fine there too. I put a virtualenv Py3.6 on my university\u2019s cluster to run some tests while writing the other changes shown here, and this appeared out of nowhere.</p>", "type": "rendered"}, "created_on": "2018-09-13T22:53:12.551835+00:00", "user": {"display_name": "Joshua Klein", "uuid": "{919f0add-304d-4b9a-8889-d2622a3dbc96}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B919f0add-304d-4b9a-8889-d2622a3dbc96%7D"}, "html": {"href": "https://bitbucket.org/%7B919f0add-304d-4b9a-8889-d2622a3dbc96%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/7d0e70bc74f783efa621a2bdd228ca22d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsJK-3.png"}}, "nickname": "mobiusklein", "type": "user", "account_id": "557058:ff82222f-afe5-4135-a1b7-8de99a00f669"}, "inline": {"to": null, "from": null, "outdated": true, "path": "pyteomics/mass/unimod.py"}, "updated_on": "2018-09-13T22:53:12.610023+00:00", "type": "pullrequest_comment", "id": 75958427}